ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1017,7 +1202,7 @@ export const RuntimePanel = fp.flow(
   const renderUpdateButton = () => {
     return <Button
       aria-label='Update'
-      disabled={!runtimeCanBeUpdated}
+      disabled={!(runtimeCanBeUpdated || pdSizeReduced)}
```

Comment:
Why is this special cased? WHy would we disable the button here?

Reply:
We won't disable the Update Button
- if a running runtime can be updated and need to be reboot or recreated
- if an unattached PD is reduced and need to be recreated

Reply:
This should feed into `runtimeCanBeUpdated` instead - I don't know why we're also consulting with this separate variable.